dagshub/upload/wrapper.py

```diff
@@ -234,14 +400,47 @@ def get_file(file: Union[str, IOBase], path=None):
             raise e
 
     def _reset_dataset(self):
+        """
+        The _reset_dataset function clears the files attribute of a Dataset object.
+        
+        
+        
+        Returns:
+            The empty list
+        
+        """
         self.files.clear()
 
     def commit(self, commit_message=DEFAULT_COMMIT_MESSAGE, *args, **kwargs):
+        """
+        The commit function is used to commit the files in the dataset.
+        It takes a commit message as an argument, which can be set to None if no message is required.
+        The function returns nothing.
+        
```

Comment:
Actually, if the message is set to `None` the server will use the default message that's defined there. A message is technically required for a git commit.